src/domain/service/note.ts

```diff
@@ -326,4 +363,75 @@ export default class NoteService {
 
     return note.publicId;
   }
+
+  /**
+   * Check if content changes are valuable enough to save currently changed note to the history
+   * @param noteId - id of the note that is currently changed
+   * @param content - updated note content
+   * @returns - boolean, true if changes are valuable enough, false otherwise
+   */
+  public async checkContentDifference(noteId: NoteInternalId, content: Note['content']): Promise<boolean> {
```

Comment:
such functions should be stored separately and covered with unit tests

Reply:
logic of this method also covered in tests, updated description